README.md

```diff
@@ -326,10 +333,19 @@ Challenge response inputs written to the file: /Users/jimzhang/iden3/AliceWonder
 
 ## Generate the Zero Knowledge Proof for the Claim Challenge
 
-We use snark.js again to generate the proof, using the inputs that have been generated in the above step.
+We use snark.js again to generate the proof, using the inputs that have been generated in the above step and callback to the verifier server for verification. `IDEN3_NAME` should be set to the issuer name.
```

Comment:
Using the issuer name for `IDEN3_NAME` will put the witness file in the issuer's directory, but this is a holder step. Is that intended?

Reply:
No. this is wrong, will fix it.

Reply:
Fixed